src/styles/mixins.scss

```diff
@@ -152,6 +149,17 @@
     .data-table__row:hover .data-table__td:nth-child(#{$nth}) {
         background-color: var(--ydb-data-table-color-hover) !important;
     }
+
+    @for $i from 0 through $nth {
```

Comment:
Stylelint refused to commit without this fix (expected rule to come before at-rule with a block)